README.md

```diff
@@ -612,8 +612,7 @@ we're building ffmpeg.
 ### Horizontal worker scaling
 
 Automated horizontal worker scaling is run via a cron job on the ganglia
-monitoring node (monitoring-master1). This uses [our
-ec2-management](https://github.com/harvard-dce/ec2-management) python script.
+monitoring node (monitoring-master1). This uses [our mo-scaler](https://github.com/harvard-dce/mo-scaler) python script.
```

Comment:
Looks good to me, and given that we do this by default it makes sense to document this way. Feel free to merge whenever.